module/x/gravity/types/interfaces.go

```diff
@@ -23,7 +23,7 @@ type EthereumSignature interface {
 type EthereumEvent interface {
 	proto.Message
 
-	GetNonce() uint64
+	GetEventNonce() uint64
```

Comment:
I personally don't like this rename, I agree with @mvid that we can use `event.GetNonce()` and is not redundant as the change proposed here `event.GetEventNonce()`